.github/workflows/release.yml

```
@@ -270,9 +395,24 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
       - name: Build & Push Image
-        env:
-          IMAGE_ORG: ${{ secrets.IMAGE_ORG}}
-        run: |
-          make docker.buildx.cstor-webhook
-          make buildx.push.cstor-webhook
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./build/cstor-webhook/cstor-webhook.Dockerfile
+          push: true
+          platforms: linux/amd64, linux/arm64
+          tags: |
+            ${{ steps.docker_meta.outputs.tags }}
+          build-args: |
+            DBUILD_DATE=${{ steps.date.outputs.DATE }}
+            DBUILD_REPO_URL=https://github.com/openebs/cstor-operators
+            DBUILD_SITE_URL=https://openebs.io
+            RELEASE_TAG=${{ env.RELEASE_TAG }}
```

```suggestion
            RELEASE_TAG=${{ env.RELEASE_TAG }}

  downstream-tagging:
    runs-on: ubuntu-latest
    needs: ['cspc-operator', 'cvc-operator', 'pool-manager', 'volume-manager', 'cstor-webhook']
    steps:
      - name: Downstream tagging
        uses: akhilerm/openebs-release-mgmt@v1.1.0
        with:
          tag-name: ${{ github.ref }}
          body: 'Release created from cstor-operators'
          repo: |
            velero-plugin
            cstor-csi
            upgrade
          # GR_TOKEN secret is the access token to perform github releases
          github-token: ${{ secrets.GR_TOKEN }}
```

@shubham14bajpai Can you take this change also into the PR as per openebs/openebs#3352, so that the downstream tagging can also be performed.

---

`GR_TOKEN` also needs to be set on the repo, so that github releases can be made.

---

Done